X/xkbcommon/build_tarballs.jl

```diff
@@ -55,17 +31,17 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency("Xorg_xorgproto_jll"),
-    Dependency("Xorg_xkeyboard_config_jll"),
-    Dependency("Xorg_libxcb_jll"),
-    Dependency("Wayland_jll"),
-    Dependency("Wayland_protocols_jll"),
     HostBuildDependency("EpollShim_jll"),
     HostBuildDependency("Wayland_jll"),
     HostBuildDependency("Wayland_protocols_jll"),
+    BuildDependency("Xorg_xorgproto_jll"),
+    Dependency("Wayland_jll"),
+    Dependency("Wayland_protocols_jll"),
+    Dependency("Xorg_libxcb_jll"),
+    Dependency("Xorg_xkeyboard_config_jll"),
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
+# We need GCC 8 because this links to XML2, which requires GCC 8
```

Comment:
Mumble mumble mumble: xml2 is not listed in the deps.  If that's a direct dependency then it should be listed explicitly, otherwise we risk another https://discourse.julialang.org/t/issues-with-librsvg-2-2-dll-on-windows-11/130201

Reply:
From what I can tell, none of the dependent packages in https://juliahub.com/ui/Packages/General/xkbcommon_jll#dependents use xkbregistry 

I built https://github.com/nhz2/xkbcommon_jll.jl with only:
```julia
dependencies = [
    BuildDependency("Xorg_xorgproto_jll"),
    Dependency("Xorg_libxcb_jll"),
    Dependency("Xorg_xkeyboard_config_jll"),
]
```

And the additional meson flags  `-Denable-docs=false -Denable-xkbregistry=false`

Reply:
Done.

Reply:
Yes, Ref:

https://github.com/xkbcommon/libxkbcommon/pull/116#issuecomment-548544519
https://github.com/xkbcommon/libxkbcommon/blob/xkbcommon-1.9.2/PACKAGING#L65-L67
https://github.com/xkbcommon/libxkbcommon/blob/xkbcommon-1.9.2/PACKAGING#L74-L76

I also searched through the dependent's source code, and couldn't find any reference to "xkbregistry".